agnpy/absorption/absorption.py

```diff
@@ -443,6 +445,113 @@ def evaluate_tau_blr_mu_s(
         )
         return (prefactor * integral).to_value("")
 
+    @staticmethod
+    def compute_integrand(cubepy_params_array, R_line, mu_s, epsilon_1, epsilon_line):
+        """
+        Computes the integrand for the gamma-gamma absorption model with
+        a spherical shell Broad Line Region (BLR).
+
+        Parameters
+        ----------
+        cubepy_params_array : array-like
+            Array containing integration parameters [mu, phi, log_l].
+        R_line : :class:`~astropy.units.Quantity`
+            Radius of the BLR spherical shell.
+        mu_s : float
+            Cosine of the angle between the blob motion and the jet axis.
+        epsilon_1 : float
+            Dimensionless energy parameter for the incoming photon.
+        epsilon_line : float
+            Dimensionless energy of the emitted line.
+
+        Returns
+        -------
+        float
+            The value of the integrand for the given parameters.
```

Comment:
it would be useful to specify in which units the integrand is specified

Reply:
Done